etc/testing/circle/workloads/metric-collector/app.py

```diff
@@ -0,0 +1,48 @@
+import io
+from google.cloud import bigquery
+import python_pachyderm
+import datetime
+from python_pachyderm.service import pps_proto
+from google.protobuf import empty_pb2
+from python_pachyderm.proto.v2.version.versionpb import version_pb2, version_pb2_grpc
+import os
+import sys
+import getopt
+from datetime import datetime
+import pprint
+
+def main():
+    
+    pachClient = python_pachyderm.Client.new_in_cluster()
+
+    # Construct a BigQuery client object.
+    client = bigquery.Client()
+
+    table = client.get_table("{}.{}.{}".format("build-release-001", "insights", "perf-tests"))
+
+    version = pachClient.get_remote_version()
+
+    version = "".join(f'{version.major}.{version.minor}.{version.micro}{version.additional}'.split())
+
+    pipelines = list(pachClient.list_pipeline())
+
+    for p in pipelines:
+
+        jobs = list(pachClient.list_job(p.pipeline.name))
+
+        for j in jobs:
+            started = float((str(j.started.seconds) + "." + str(j.started.nanos)))
+            finished = float((str(j.finished.seconds) + "." + str(j.finished.nanos)))
```

Comment:
I'm not sure this is completely right. For example, if nanos is 10(an int32, so no leading 0s), and seconds is 50 you get 50.10 seconds, but .10 seconds is way more than 10 nanos. The protobuf timestamp data type has an example with Python's Timestamp class which seems more reliable [https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.Timestamp](https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.Timestamp)

Reply:
Actually it looks like JobInfo has stats.processTime(as well as upload and download time). You could potentially just use those. Up to you if you want to do this calculation instead, either way works for me.

Reply:
@djanicekpach good catch. actually after thinking a bit more, I want make this collector super simple. so i'm going to simply collect and do calculations/visualization/stats elsewhere. I want to focus back on getting the env creations back to even start running this.